exercises/Toluwalase_Agoro/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,184 @@
+
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include <string>
+#include <iomanip>
+using namespace std;
+
+
+void toLower(string &stringName) {
+	/* Function accepts a string and modifies string
+	by making it lowercase and returns nothing
+	*/
+	string PlaceHolderString{  };
+	for (char Letter : stringName) 
+		PlaceHolderString += tolower(Letter);
+	
+	stringName = PlaceHolderString;
+
+}
+
+
+void removePunctuation(string &stringName) {
+	/* Function accepts a string a modifies string by
+	removing any occurence of ".,?'"!():" within word 
+	and returns nothing
+	*/
+	for (int i = 0; i < stringName.size();) {
+		char a{ stringName[i] };
+		if (stringName[i] == '"'|| stringName[i] == '!' || stringName[i] == '(' 
+			|| stringName[i] == ')' || stringName[i] == ':' || stringName[i] == '\'' 
+			|| stringName[i] == ',' || stringName[i] == '?' || stringName[i] == '.') 
+		{
+			stringName.erase(i, 1);
+			i--;
+		}
+		i++;
+	}
+}
+
+
+string getWord(ifstream &stream) {
+	/* Function accept an input stream object and return a string from 
+	input stream by reading stream 1 character at a time and concatenating
+	it to a string until a space ,newline or '-'  character.
+	*/
+	string OutputString{};
+	char InputCharacter{};
+
+	while (true) {
+		stream.get(InputCharacter);
+		if (InputCharacter == '-' || InputCharacter == ' ' || InputCharacter == '\n'||stream.eof())
+			break;
+		OutputString += InputCharacter;
+	}
+	return OutputString;
+}
+
+
+int findString(const string &SearchString, const vector <string> &SearchList) {
```

Comment:
There's a "find" function in the standard library that does this for you so I think you could remove the findString function eg
auto it = std::find(VectorOfWords.begin(),VectorOfWords.end(),HolderString)
if (HolderString.size() > 4 && it==VectorOfWords.end()){
}